components/coachmark/index.css

```diff
@@ -119,7 +119,7 @@
 
 /* Wrapper element around the nested action menu sub-component */
 .spectrum-CoachMark-action-menu {
-	--mod-popover-content-area-spacing: var(--spectrum-popover-content-area-spacing);
+	--mod-popover-content-area-spacing: var(--spectrum-popover-edge-to-content-area);
```

Comment:
The variable name before is defined inside `.spectrum-Popover` and so not available to this component. I moved instead to using the global token value that the `.spectrum-Popover` variable is mapped to.